### PR TITLE
[+] Added French Language

### PR DESCRIPTION
**Pull request by ZeProf2Code**
Hello, I added the french language.

### CODE REVIEW

**Comment by lancepioch** [comment]
Will review this weekend

File: resources/lang/fr/validation.php [hunk 1]
```diff
@@ -1,66 +1,105 @@
 <?php
```
Comment (DaneEveritt) [inline resources/lang/fr/validation.php]: Why was this file converted back to english?

**Comment by ZeProf2Code** [comment]
I'm translating it

**Comment by ZeProf2Code** [comment]
@DaneEveritt It's ready :)

**Pull request merged**